sr_object_tracking/launch/tracking.launch

```diff
@@ -31,9 +23,15 @@
     <param name="color" value="$(arg color)" />
   </node>
 
-  <node name="viewer" pkg="sr_gui_servoing" type="displaying.py" >
-    <param name="saturation" value="$(arg saturation_min)" />
-    <param name="color" value="$(arg color)" />
-  </node>
+  <group if="$(arg display)">
+    <node name="viewer" pkg="sr_gui_servoing" type="displaying.py" >
+      <param name="saturation" value="$(arg saturation_min)" />
+      <param name="color" value="$(arg color)" />
+    </node>
+  </group>
+
+  <group if="$(arg camera)">
+    <include file="$(find sr_object_tracking)/launch/camera.launch"/>
```

Comment:
you'll need to pass a kinect argument through as well (set it in tracking.launch, pass it to camera.launch)